tests/test_downloadermiddleware_redirect.py

```diff
@@ -42,6 +42,10 @@ def _test(method):
         _test('POST')
         _test('HEAD')
 
+        _test('GET', status=308)
+        _test('POST', status=308)
+        _test('HEAD', status=308)
```

Comment:
tests are fine but looks like a good opportunity to add tests for 307 status too